core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -99,6 +113,62 @@ angular.module('oppia').directive('adminMiscTab', [
           });
         };
 
+        const populateExplorationStatsRegenerationCsvResult = function(expIds) {
+          if (expIds.length === 0) {
+            let csvContent = explorationStatsRegenerationUtilityCsvRows.map(
+              row => row.join(',')).join('\n');
+            ctrl.csvUri = $sce.trustAsResourceUrl(
+              window.URL.createObjectURL(
+                new Blob([csvContent], { type: 'text/csv' })));
+            ctrl.csvFilename = (
+              `missing-exp-stats-output-${(new Date()).getTime()}.csv`);
+            ctrl.missingExplorationStatsRegenerationMessage = (
+              'Process complete. Please download the CSV output.');
+            return;
+          }
+          let expIdToRegenerate = expIds.pop();
+          ctrl.missingExplorationStatsRegenerationMessage = (
+            `Regenerating stats for Exploration id ${expIdToRegenerate}` +
+            '. Please wait.');
+          $http.post(ADMIN_HANDLER_URL, {
```

Comment:
We are moving all admin-page-related http requests to backend-api.service ([active PR](https://github.com/oppia/oppia/pull/11463/files#diff-08e907c6ac1d419e962d2e60ae8f0de8ecd99f7504397887107ae7d893aac610R222)), can we move this too?

Reply:
Based on offline discussion, this will be moved to the `admin-backend-api.service` in #11463.